src/database/seeders/PermissionSeeder.php

```diff
@@ -0,0 +1,104 @@
+
+class PermissionSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     */
+    public function run(): void
+    {
+        // Create all roles
+        $superAdmin = Role::create(['name' => Roles::SUPERADMIN]);
+        $colleague = Role::create(['name' => Roles::COLLEAGUE]);
+        $customer = Role::create(['name' => Roles::CUSTOMER]);
+
+        // Create all permissions
+
+        $borrow = Permission::create(['name' => Permissions::BORROW]);
+        $reserve = Permission::create(['name' => Permissions::RESERVE]);
+        $return = Permission::create(['name' => Permissions::RETURN]);
+        $loginManage = Permission::create(['name' => Permissions::LOGIN_MANAGE]);
+
+        $viewLibraryPass = Permission::create(['name' => Permissions::VIEW_LIBRARY_PASS]);
+        $createLibraryPass = Permission::create(['name' => Permissions::CREATE_LIBRARY_PASS]);
+        $editLibraryPass = Permission::create(['name' => Permissions::EDIT_LIBRARY_PASS]);
+        $deleteLibraryPass = Permission::create(['name' => Permissions::DELETE_LIBRARY_PASS]);
+
+        $viewItem = Permission::create(['name' => Permissions::VIEW_ITEM]);
+        $createItem = Permission::create(['name' => Permissions::CREATE_ITEM]);
+        $editItem = Permission::create(['name' => Permissions::EDIT_ITEM]);
+        $deleteItem = Permission::create(['name' => Permissions::DELETE_ITEM]);
+
+        $viewUser = Permission::create(['name' => Permissions::VIEW_USER]);
+        $createUser = Permission::create(['name' => Permissions::CREATE_USER]);
+        $editUser = Permission::create(['name' => Permissions::EDIT_USER]);
+        $deleteUser = Permission::create(['name' => Permissions::DELETE_USER]);
+
+        $viewGrant = Permission::create(['name' => Permissions::VIEW_GRANT]);
+        $createGrant = Permission::create(['name' => Permissions::CREATE_GRANT]);
+        $editGrant = Permission::create(['name' => Permissions::EDIT_GRANT]);
+        $deleteGrant = Permission::create(['name' => Permissions::DELETE_GRANT]);
+
+        $viewReservation = Permission::create(['name' => Permissions::VIEW_RESERVATION]);
+        $createReservation = Permission::create(['name' => Permissions::CREATE_RESERVATION]);
+        $editReservation = Permission::create(['name' => Permissions::EDIT_RESERVATION]);
+        $deleteReservation = Permission::create(['name' => Permissions::DELETE_RESERVATION]);
+
+        $viewPayment = Permission::create(['name' => Permissions::VIEW_PAYMENT]);
+        $createPayment = Permission::create(['name' => Permissions::CREATE_PAYMENT]);
+        $editPayment = Permission::create(['name' => Permissions::EDIT_PAYMENT]);
+
+        $viewFine = Permission::create(['name' => Permissions::VIEW_FINE]);
+        $editFine = Permission::create(['name' => Permissions::EDIT_FINE]);
+
+
+
+        // -- CUSTOMER
+        $customer->givePermissionTo($borrow);
+        $customer->givePermissionTo($reserve);
+        $customer->givePermissionTo($return);
```

Comment:
Can a customer view what he is borrowing?
Do we want a customer to be able to cancel a reservation?
Can a customer see what fines he has, and can he pay them?

Reply:
1 - FF voor je zien dat de user al in de terminal zit. Ik kan voor die basis rechten wel permissions maken maar willen we dat wel? Ik zie geen reden om die dingen te beperken bij mensen. 

2  - Niet besproken dus ook niet geimplementeerd. Dit hoort een review te zijn niet om nieuwe features te bedenken.

3 - Zie 1

4 - Praat gwn nederlands anders ga ik oprecht alles in het duits hier neer zetten.

Reply:
Chris du sprichst nur Deutsch, nicht Englisch und nicht Niederländisch

Reply:
Okay, gut, dann lass uns Deutsch sprechen